keps/349-all-or-nothing/README.md

```diff
@@ -225,6 +250,28 @@ type WaitForPodsReady struct {
 	// RequeuingStrategy defines the strategy for requeuing a Workload.
 	// +optional
 	RequeuingStrategy *RequeuingStrategy `json:"requeuingStrategy,omitempty"`
+
+	// WaitForAdmissionChecks defines configuration for the WaitForAdmissionChecks feature,
+	// which is used to ensure that specific AdmissionCheck is ready within the specified time.
+	// If there is no specified configuration for an AdmissionCheck, there is no timeout for it.
+	// +listType=map
+	WaitForAdmissionChecks []WaitForAdmissionChecks
```

Comment:
It looks a bit weird here, especially given that there is RequeuingStrategy is present above and in WaitForAdmissionChecks.